content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -36,3 +42,7 @@ Corda supports the following `SignatureSpecs` (signature schemes) for creating `
 * GOST3411withGOST3410
 
 Use `SignatureSpecService` to retrieve the `SignatureSpec`.
+
+Initially the above list of signature spec was available to users. However, they could be passing the wrong signature spec for a signing key type (for example, an RSA key with an SHA256withECDSA), which would lead to an error when attempting to generate the signature.
+
+Instead, `SignatureSpecService` was introduced. It takes in a key (and a digest algorithm optionally) and returns a default signature spec for those.
```

Comment:
These two lines read like a Release Note and I don't think belong here.

`SignatureSpecService` ensures that you do not pass the wrong signature spec for a signing-key type. It takes in a key and, optionally, a digest algorithm, and returns the appropriate default signature spec.